msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -0,0 +1,432 @@
+
+    1. Select **Create a new Teams app** option to create an app using Teams Toolkit.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/create-new-app.png" alt-text="Wizard starts for creating a new project" border="true":::
+
+    1. Ensure that **Notification Bot** is selected as the capability that you want to build in your app. 
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/select-notification-bot.png" alt-text="Select App Capability" border="true":::
+
+    1. Select **JavaScript** as the programming language.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/select-script.png" alt-text="Screenshot showing how to select the programming language." border="true":::
+
+    1. Select **HTTP Trigger Restify Server** as the trigger. 
+    1. Select **OK**.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/select-trigger.png" alt-text="Screenshot showing how to select the programming language." border="true":::  
+
+    1. Enter a suitable name for your app, such as mynotificationbot as the application name. 
+    1. Select **Enter**.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/type-notfication-name.png" alt-text="Screenshot showing where to enter the app name." border="true":::
+
+       > [!NOTE]
+       > Ensure that you use only alphanumeric characters in the application name.
+
+        The notification bot app is created in a few seconds.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/my-notification-bot.png" alt-text="Screenshot showing the app created." border="true":::
+
+        After the app is created, Teams Toolkit displays the location where the project workspace is created. This folder is the default location specified in Visual Studio Code settings. It gives options to **Change Location** or **Local Debug**.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/location-change.png" alt-text="Location message"border="true":::
+
+        > [!NOTE]
+        > For this tutorial, don't use Change Location and Local Debug options. If you choose to change the location for workspace, it's reflected only for projects you create after reloading Visual Studio Code. No change is made to the location of existing projects.
+    
+    1. Select **Run and Debug** :::image type="icon" source="../msteams-platform/assets/images/teams-toolkit-v2/run-debug-icon.png"::: icon from the Visual Studio Code sidebar.
+
+    1. Select **Debug(Edge)** or **Debug(Chrome)** from the dropdown list.
+
+    1. Select **Play** to start debugging.
+
+        :::image type="content" source="../msteams-platform/assets/images/sbs-notification-bot/debug-app.png" alt-text="App Debugging"border="true"::: 
+
```

Comment:
Before people can see the Teams window, if it is the first time, they will be prompted to sign in with their m365 account. It will be better to let them know
1. they need to sign in with work/school account
2. sign in with the account in the tenant with sideloading permission, otherwise the "Add" button in the next step will be disabled.

Reply:
It is already mentioned in the Prerequisites step.